google/cloud/spanner_dbapi/connection.py

```diff
@@ -122,19 +144,16 @@ def close(self):
 
         The connection will be unusable from this point forward.
         """
-        self.rollback()
         self.__dbhandle = None
         self.is_closed = True
 
     def commit(self):
-        self._raise_if_closed()
-
-        self.run_prior_DDL_statements()
+        """Commit all the pending transactions."""
+        warnings.warn(AUTOCOMMIT_MODE_WARNING, UserWarning, stacklevel=2)
 
     def rollback(self):
-        self._raise_if_closed()
-
-        # TODO: to be added.
+        """Rollback all the pending transactions."""
+        warnings.warn(AUTOCOMMIT_MODE_WARNING, UserWarning, stacklevel=2)
```

Comment:
I've used just `warnings` shots as raising an exception in these methods will break a lot of Django tests.